plugin/clouddns/gcp.go

```diff
@@ -23,10 +27,16 @@ func (c gcpClient) zoneExists(projectName, hostedZoneName string) error {
 
 // listRRSets is a wrapper method around `gcp.Service.ResourceRecordSets.List`
 // it fetches and returns the record sets for a hosted zone.
-func (c gcpClient) listRRSets(projectName, hostedZoneName string) (*gcp.ResourceRecordSetsListResponse, error) {
-	rr, err := c.ResourceRecordSets.List(projectName, hostedZoneName).Do()
-	if err != nil {
+func (c gcpClient) listRRSets(ctx context.Context, projectName, hostedZoneName string) (*gcp.ResourceRecordSetsListResponse, error) {
+	req := c.ResourceRecordSets.List(projectName, hostedZoneName)
```

Comment:
No longer need to check for error here?

Reply:
👍  correct, we should validate here, I'll add & push soon.

Reply:
Yes, it's be checked while listing Pages